### PR TITLE
[skip ci] PEP8ify expr_with_intlimits.py

### PR DESCRIPTION
**Pull request by jremes-foss**
Harmonized the indentations and added extra spaces where requires.

Thank you.

### CODE REVIEW

File: sympy/concrete/expr_with_intlimits.py [hunk 1]
```diff
@@ -23,7 +26,7 @@ def change_index(self, var, trafo, newvar=None):
         Usage
         =====
 
-        ``change_index(expr, var, trafo, newvar=None)`` where ``var`` specifies the
+        ``change_index(expr, var, trafo, newvar = None)`` where ``var`` specifies the
```
Comment (toolforger) [inline sympy/concrete/expr_with_intlimits.py]: PEP8 mandates blanks around `=` for assignments but not for keyword arguments and default parameters.

File: sympy/concrete/expr_with_intlimits.py [hunk 2]
```diff
@@ -113,18 +116,18 @@ def change_index(self, var, trafo, newvar=None):
                 beta = p.coeff_monomial(S.One)
                 if alpha.is_number:
                     if alpha == S.One:
-                        limits.append((newvar, alpha*limit[1] + beta, alpha*limit[2] + beta))
+                        limits.append((newvar, alpha * limit[1] + beta, alpha * limit[2] + beta))
```
Comment (toolforger) [inline sympy/concrete/expr_with_intlimits.py]: Here not having blanks around `*` is actually okay, it's indicating operator precedence.
(Also, PEP8 gives `hypot2 = x*x + y*y` as an example for okay code.)

File: sympy/concrete/expr_with_intlimits.py [hunk 3]
```diff
@@ -113,18 +116,18 @@ def change_index(self, var, trafo, newvar=None):
...
                     elif alpha == S.NegativeOne:
-                        limits.append((newvar, alpha*limit[2] + beta, alpha*limit[1] + beta))
+                        limits.append((newvar, alpha * limit[2] + beta, alpha * limit[1] + beta))
```
Comment (toolforger) [inline sympy/concrete/expr_with_intlimits.py]: See above about spaces about operators. (More similar places below, won't annotate each one of them.)

**Comment by Newman101** [comment]
Closing this pull request, thanks for the information.

**Pull request closed without merging**